Source/WebCore/platform/text/hyphen/HyphenationLibHyphen.cpp

```diff
@@ -76,11 +76,13 @@ static void scanDirectoryForDictionaries(const char* directoryPath, UncheckedKey
             continue;
 
         auto filePath = FileSystem::pathByAppendingComponent(directoryPathString, fileName);
-        char normalizedPath[PATH_MAX];
-        if (!realpath(FileSystem::fileSystemRepresentation(filePath).data(), normalizedPath))
+        char* normalizedPath = realpath(FileSystem::fileSystemRepresentation(filePath).data(), nullptr);
```

Comment:
Here you should simply use `FileSystem::realPath`.

The JSC API test calls it directly because it doesn't use WebKit internals, and then FilesystemPlayStation.cpp does so to implement `FileSystem::realPath`, but everywhere else can safely use it.

Reply:
ok, done so